File: OnlineShopUI/OnlineShop/src/app/model/requests/add-payment-request.ts
export class AddPaymentRequest {
    paymentId: number;
    nameOnCard: string;
    cardNumber: string;
    securityCode: string;
    expDate: string;
    cardTypeId: number;
    accountId: number;

    constructor(paymentId: number, nameOnCard: string, cardNumber: string, securityCode: string, expDate: string, cardTypeId: number, accountId: number){
        this.paymentId = paymentId;
        this.nameOnCard = nameOnCard;
        this.cardNumber = cardNumber;
        this.securityCode = securityCode;
        this.expDate = expDate;
        this.cardTypeId = cardTypeId;
        this.accountId = accountId;
    }
}

File: OnlineShopAPI/PaymentInfoMicroservice/PaymentInfo.Api/Models/AddPaymentRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PaymentInfo.Api.Models
{
    public class AddPaymentRequest
    {
        public long PaymentId { get; set; }

        public string NameOnCard { get; set; }

        public string CardNumber { get; set; }

        public string SecurityCode { get; set; }

        public string ExpDate { get; set; }

        public long CardTypeId { get; set; }

        public long AccountId { get; set; }
    }
}
